src/libsync/discovery.cpp

```diff
@@ -516,6 +539,7 @@ void ProcessDirectoryJob::processFileAnalyzeRemoteInfo(
         const bool isVirtualE2EePlaceholder = isDbEntryAnE2EePlaceholder && serverEntry.size >= Constants::e2EeTagSize;
         const qint64 sizeOnServer = isVirtualE2EePlaceholder ? serverEntry.size - Constants::e2EeTagSize : serverEntry.size;
         const bool metaDataSizeNeedsUpdateForE2EeFilePlaceholder = isVirtualE2EePlaceholder && dbEntry._fileSize == serverEntry.size;
+        const bool serverEntryLockedAsBool = serverEntry.locked == SyncFileItem::LockStatus::LockedItem;
```

Comment:
@claucambra I don't think the `AsBool` part is necessary. Could just be `const auto isServerEntryLocked = serverEntry.locked == SyncFileItem::LockStatus::LockedItem;`, then it becomes obvious from the naming.

Reply:
This is cleaner, thanks, changed